src/file-1.xoo

```diff
@@ -0,0 +1,3 @@
+// dummy line 1
+// dummy line 2
+// violate HasTag rule: xoo on line 3
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Has Tag ([xoo:HasTag](http://127.0.0.1:34395/organizations/org1/rules#rule_key=xoo%3AHasTag))

[See it in SonarCloud](http://127.0.0.1:34395/project/issues?id=sample&pullRequest=5615&issues=AWcspGj3APl1pxP08xGK&open=AWcspGj3APl1pxP08xGK)

[ISSUE_KEY]: AWcspGj3APl1pxP08xGK